package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "employee-module-using-typescript",
+  "version": "1.0.0",
+  "description": "rest-api for an employee module",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon"
+  },
+  "author": "me",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/bcrypt": "^5.0.0",
```

Comment:
separate dependencies and devDependencies list

Reply:
Done. Could you please check if I have separated them correctly?